### ISSUE TITLE
404 on events page because of missing https in airtable entries

### ISSUE THREAD

**Issue by muescha** [gatsbyjs/gatsby#20184]
## Description

there are missing `https://` in the connected airtable in the events table 
for listing events on page: https://www.gatsbyjs.org/contributing/events/

### option a

fixing data in the airtable

### option b

add a check for the `https` to the link in `www/src/components/events/event.js`

#### Link 1
https://www.gatsbyjs.org/contributing/events/karunyahacks.org

BULD HACKATHON — November 17, 2019 (Coimbatore, Tamil Nadu, India)

#### Link 2

https://www.gatsbyjs.org/contributing/events/nosk.ncit.edu.np

Software Freedom Day — September 21, 2019 (Balkumari, Lalitpur, Nepal)

#### Link 3

https://www.gatsbyjs.org/contributing/events/wizathon.in

Software Freedom Day — September 21, 2019 (Balkumari, Lalitpur, Nepal)

#### Link 4

https://www.gatsbyjs.org/contributing/events/www.startupweekend.com

Startup Weekend Youth Edition — November 15, 2019 (Dubai,United Arab Emirates)

### Steps to reproduce

go to page: https://www.gatsbyjs.org/contributing/events/

### Expected result

no error 

### Actual result

error 404

### related Issues

#19267 Add link check To Gatsby Docs

**Comment by muescha** [comment]
more links broken on this site:

- https://www.meetup.com/es-ES/Rockstar-JavaScript-Barcelona/events/260615768
- https://www.eventbrite.com/e/state-of-drupal-9-using-drupal-and-gatsby-wunder-tech-meetup-tickets-62198670899
- http://arcs.ieeecsvit.com/%20(We%20are%20currently%20working%20on%20the%20present%20years's%20website)

**Comment by muescha** [comment]
someone here with access to the airtable table to fix the entries there?

**Comment by muescha** [comment]
there are some new broken links in beetween:

https://www.meetup.com/es-ES/Rockstar-JavaScript-Barcelona/events/260615768
https://www.gatsbyjs.org/contributing/events/xperion.ieeesbnssce.org
https://www.gatsbyjs.org/contributing/events/www.startupweekend.com
https://www.gatsbyjs.org/contributing/events/www.hackbmu.com
https://www.gatsbyjs.org/contributing/events/wizathon.in
https://www.gatsbyjs.org/contributing/events/nosk.ncit.edu.np
https://www.gatsbyjs.org/contributing/events/karunyahacks.org
https://www.gatsbyjs.org/contributing/events/hack.macehub.in
https://www.eventbrite.com/e/state-of-drupal-9-using-drupal-and-gatsby-wunder-tech-meetup-tickets-62198670899
https://www.eventbrite.com/e/84398059869
https://trailblazercommunitygroups.com/e/m2mjcq
http://arcs.ieeecsvit.com/%20(We%20are%20currently%20working%20on%20the%20present%20years's%20website)


ping to @gatsbyjs/core @gatsbyjs/learning

**Comment by gillkyle** [comment]
Thanks for the research investigating this and the ping @muescha! I'll go in now and update the links missing the `http://` protocol on Airtable. I do like the idea of finding a better check for urls!

**Comment by muescha** [comment]
this link can also fixed in the airtable:
http://arcs.ieeecsvit.com/%20(We%20are%20currently%20working%20on%20the%20present%20years's%20website) to http://arcs.ieeecsvit.com/

**Comment by gillkyle** [comment]
I updated the http://arcs.ieeecsvit.com site and removed entries for urls that were resulting in 404s if they were already past so it doesn't negatively affect the page's SEO. 

I also updated the airtable request form with more context to instruct users to include the scheme (http or https) as well as an example so that hopefully most won't input a broken link in the future. 

I think those measures should allow us to close this for now. If it's still coming up in the future we can enforce better validation but for now I think this gets the job done. Thanks for the help @muescha!
